### PR TITLE
SafeConfigParser is now called as ConfigParser in P3

### PR DESCRIPTION
**Pull request by nehemiah-hivery**
Simple and straightforward import, 14 tests are passing before and after this commit under P2 and P3 respectively.

### CODE REVIEW

File: dynamodb_config_store/__init__.py [hunk 1]
```diff
@@ -33,7 +33,12 @@
 """
 import os.path
 import time
-from ConfigParser import SafeConfigParser
+
```
Comment (sebdah) [inline dynamodb_config_store/__init__.py]: Please remove this blank line and group the imports from the standard library together.

File: setup.py [hunk 2]
```diff
@@ -1,7 +1,12 @@
 """ Setup script for PyPI """
 import os
 from setuptools import setup
-from ConfigParser import SafeConfigParser
+
```
Comment (sebdah) [inline setup.py]: Please remove this blank line and group the imports from the standard library together.

**Review by sebdah — CHANGES_REQUESTED** [review]
Thanks for the PR. It looks great, just had a small comment.

**Comment by nehemiah-hivery** [comment]
done

**Review by sebdah — APPROVED** [review]

**Pull request merged**